articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,372 @@
+import LinkedAccountItem from './LinkedAccountItem'
+import AuthService from 'utils/AuthService'
+import styles from './styles.module.css'
+
+export class LinkedAccountsList extends React.Component {
+  static propTypes = {
+    auth: T.instanceOf(AuthService),
+    profile: T.object
+  }
+
+  render(){
+    const { profile, auth } = this.props
+    let items = []
+    if (profile) {
+      items = profile.identities.map(identity => {
+        return (<LinkedAccountItem {...this.props} identity={identity} />)
+      })
+    }
+
+    return (
+      <div className={styles.root}>
+        <h3>Linked Accounts</h3>
+        <ListGroup>{items}</ListGroup>
+      </div>
+    )
+  }
+}
+
+export default LinkedAccountsList;
+```
+
+Bascially `LinkedAccountsList` renders a container and then one `LinkedAccountItem` for each identity in current user profile. It's expecting `profile` and `auth` as props, which we'll the sent by its parent, the `Home` component:
+
+```javascript
+/* ===== ./src/views/Main/Home/Home.js ===== */
+import React, { PropTypes as T } from 'react'
+import {Row, Col, Thumbnail, Button} from 'react-bootstrap'
+import AuthService from 'utils/AuthService'
+import LinkedAccountsList from 'components/LinkedAccount/LinkedAccountsList'
+import styles from './styles.module.css'
+
+export class Home extends React.Component {
+  static contextTypes = {
+    router: T.object
+  }
+
+  static propTypes = {
+    auth: T.instanceOf(AuthService)
+  }
+
+  constructor(props, context) {
+    super(props, context)
+    this.state = {
+      profile: props.auth.getProfile()
+    }
+    props.auth.on('profile_updated', (newProfile) => {
+      this.setState({profile: newProfile})
+    })
+  }
+
+  logout(){
+    this.props.auth.logout()
+    this.context.router.push('/login');
+  }
+
+  render(){
+    const { profile } = this.state
+    return (
+      <div>
+        <h2 className={styles.pageTitle}>Home</h2>
+        <Row>
+          <Col md={2} mdOffset={4} className={styles.pane}>
+            <Thumbnail src={profile.picture}>
+              <p>Welcome {profile.name}!</p>
+              <p>
+                <Button bsStyle="default" onClick={this.logout.bind(this)}>Logout</Button>
+              </p>
+            </Thumbnail>
+          </Col>
+          <Col md={4} className={styles.pane}>
+            <LinkedAccountsList profile={profile} auth={this.props.auth}></LinkedAccountsList>
+          </Col>
+        </Row>
+      </div>
+    )
+  }
+}
+
+export default Home;
+```
+
+Notice that `Home` is updated to render a left column with some profile info like name, avatar and a button to `logout`. The right columns is reserved to render the `LinkedAccountsList`. If you run the application you should see the new home page after a successful login, and the __Linked Accounts__ list shows only the main account. In the next topic we show how to add a button to link an account from an external provider.
+
+## 2. Linking Accounts
+
```

Comment:
@flaviogranero 
Please update the URL to `/api/management/v2#!/Users/post_identities`. We always use relative URLs for internal `auth0.com/docs` links.